daft/expressions/expressions.py

```diff
@@ -860,8 +985,11 @@ def not_nan(self) -> Expression:
 
         Example:
             >>> # [1., None, NaN] -> [True, None, False]
+            >>> df = daft.from_pydict({"data": [1., None, nan]})
+            >>> df = df.select(df["data"].not_nan())
+            >>> df.collect()
             >>> col("x").not_nan()
-
+            #FLAGGED
```

Comment:
Can you try running the code and put the output in here?

Reply:
df = daft.from_pydict({"x": [1., None, nan]})
            >>> df = df.select(df["x"].not_nan())
            >>> df.collect()

AttributeError: 'Expression' object has no attribute 'not_nan'